src/transformStateWithClones.js

```diff
@@ -64,7 +64,41 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const newState = {...state};
+  let statesHistory  = [];
+  for (let i = 0; i < transforms.length; i++) {
+    const { properties } = transforms[i];
```

Comment:
чому не створити зміну і для `operation`? 
```suggestion
    const { properties, operation } = transforms[i];
```